database/migrations/2021_07_28_102443_plugins_add_version_and_settings.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class PluginsAddVersionAndSettings extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('plugins', function (Blueprint $table) {
+            $table->integer('version')->default(1);
+            $table->json('settings')->nullable();
```

Comment:
Guessing this field type could be the problem?

Reply:
It is, it uses utf8mb4_bin collation, which is valid. But I think I messed up my fix for that :)